lib/ansible/modules/lineinfile.py

```diff
@@ -44,6 +44,18 @@
     type: str
     aliases: [ regex ]
     version_added: '1.7'
+  literal:
```

Comment:
This should probably be called `literal_search` to better communicate it is used for searching.
```suggestion
  literal_search:
```

Reply:
@samdoran this change affects to all the documentation and `argument_spec`. I think is better to keep the `literal` name and document properly. What do you think?

Reply:
Let's go for `search_string`, 😆 . I will change all the references and retest. Thank you!